src/winetricks

```diff
@@ -8215,6 +8215,69 @@ load_dxvk()
 
 #----------------------------------------------------------------
 
+# $1 - dxvk-nvapi archive name (required)
+# $2 - minimum Wine version (required)
+# $3 - nvapi,[nvapi64] (required)
+helper_dxvk_nvapi()
+{
+    _W_package_archive="${1}"
+    _W_min_wine_version="${2}"
+    _W_dll_overrides="$(echo "${3}" | sed 's/,/ /g')"
+    # dxvk-nvapi repository, for (partial) NVAPI support
+    _W_repository="jp7677/dxvk-nvapi"
+
+    _W_supported_overrides="nvapi nvapi64"
+    _W_invalid_overrides="$(echo "${_W_dll_overrides}" | awk -vvalid_overrides_regex="$(echo "${_W_supported_overrides}" | sed 's/ /|/g')" '{ gsub(valid_overrides_regex,""); sub("[ ]*",""); print $0 }')"
+    if [ "${_W_invalid_overrides}" != "" ]; then
+        w_die "parameter (4) unsupported dll override: '${_W_invalid_overrides}' ; supported dll overrides: ${_W_supported_overrides}"
+    fi
+
+    _W_package_dir="${_W_package_archive%.tar.gz}"
+    _W_package_version="v${_W_package_dir#*-v}"
+    w_warn "Please refer to ${_W_repository#*/} version ${_W_package_version} release notes... See: https://github.com/${_W_repository}/releases/tag/${_W_package_version}"
```

Comment:
Why the difference from dxvk? They end up with the same output, but dxvk_nvapi sets the "v" prefix in the variable assignment, while dxvk sets it in the warning itself?

Reply:
To clarify, it's okay if things differ. I wanted to know why. Ideally, the code should be shared as much as possible, but differences happen.

That said, changes to dxvk itself should be avoided (unless it's a bug fix, or a change to _share_ code).

Reply:
The only change I made to DXVK was how the package version variable collected the version tag; the value is only used for referencing the release notes here, so... what's the issue?